### DISCUSSION TITLE
kpanda/intro/features/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1985] [Announcements]
# kpanda/intro/features/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/intro/features/

**Comment by KubeKyrie** [comment]
服务与路由，我们支持的service类型应该还包括LoadBalancer：负载均衡

**Reply by lemontrees123** [reply]
已更新
